comms/discovery/schema/schema.go

```diff
@@ -200,6 +201,33 @@ type ChatInvite struct {
 	UserID     string `json:"user_id"`
 }
 
+type ChatBlastBase struct {
```

Comment:
do we need this?

Reply:
quicktype will just generate structs 1:1 to match ts file... but it's safe to ignore.